bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -124,7 +124,7 @@ public void textBoxKeyPress(final TextBoxWidget tbwe) {
 
         final SPInstObsComp inst = getContextInstrumentDataObject();
         _w.newMenu.setEnabled(enabled && inst != null);
-        _w.tag.setEnabled(enabled && !selectionIsBasePosition());
+        _w.tag.setEnabled(enabled && selectedGroup().isEmpty() && !selectionIsAutoTarget() && !selectionIsBasePosition());
```

Comment:
To maybe answer my own question, I guess we're trying to keep it disabled when a group (as opposed to a target) is selected?